facade/nodeFacade.go

```diff
@@ -337,6 +337,41 @@ func (nf *nodeFacade) GetAccount(address string, options apiData.AccountQueryOpt
 	return accountResponse, blockInfo, nil
 }
 
+// TODO: refactor GetAccounts. Make AccountRepository/AccountsAdapterAPI able to return the entire accounts adapter in order to
+// only fetch it once, and then re-use it for getting all the accounts.
+// Currently, there is no improvement in changing the existing mechanism (GetAccount -> GetAccount*s*) since it would
+// still mean the recreation of the trie for each of the account
```

Comment:
It should be _somehow_ all right, though since the `latest recreated trie` is cached.

Reply:
Plus, there aren't `N` HTTP requests anymore, which is a nice improvement by itself.

Reply:
removed todo